x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/components/agent_activity_flyout.tsx

```diff
@@ -243,6 +243,11 @@ const actionNames: {
     completedText: 'force unenrolled',
     cancelledText: 'force unenrollment',
   },
+  UPDATE_TAGS: {
+    inProgressText: 'Updating tags of',
```

Comment:
Just curious, are these strings translated? Is it something that we should do in this context?

Reply:
using them in a `FormattedMessage` component [here](https://github.com/elastic/kibana/blob/c4a4d5cf3a8be53cd009b4c3b80ba3356d01d17e/x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/components/agent_activity_flyout.tsx#L268-L270), though it might be best to translate like this. The challenge is that there is a combination of statuses and action types, so writing them out separately would create about 20 strings.

Reply:
Thanks for clarifying. I think that this could be handled in a separate ticket, as it's potentially so complex.